alarm/utils.py

```diff
@@ -33,3 +30,128 @@ def any_of_trade_pair_thresholds_is_broken(trade_pair):
             return True
 
     return False
+
+
+def refresh_thresholds_brakes_from_recent_candles_update(trade_pair):
+    thresholds = Threshold.objects.filter(trade_pair=trade_pair)
+    for threshold in Threshold.objects.filter(trade_pair=trade_pair):
+        if ThresholdBrake.objects.filter(threshold=threshold).exists():
+            continue
+    for threshold in thresholds:
+        try:
+            latest_threshold_brake = ThresholdBrake.objects.filter(threshold__trade_pair=trade_pair).latest(
+                'happened_at')
+            if latest_threshold_brake:
+                return False
+        except ThresholdBrake.DoesNotExist:
+            pass
+
+        ThresholdBrake.objects.create(threshold=threshold)
+
+
+def create_message_about_threshold_break():
```

Comment:
How will I get a message from shell? How will I get a message for an SMS with this function?
The function is coupled to an implementation with calls, it is not orthogonal.

Here should be 1 function: to have a barebones text.
Than in a function, which makes a call, you will wrap a text to what's needed for Twilio.
Again, see https://docs.google.com/document/d/1caqlO-Jkaqz9cxljMHuv7A0PGbHieP6m5XE_uy1iARY/edit. How will I get a text from a shell?

Reply:
Next, let's say I want to get a message for a specific threshold. How will I do that? No way, this function returns everything. It does too much. 

Single responsibility is to generate a message for a threshold. That should be a single function. Another function to gather threshold broken messages and combine them.

Reply:
"_Single responsibility is to generate a message for a threshold. That should be a single function. Another function to gather threshold broken messages and combine them._" - I think it's done

but I don't quite understand the idea of "getting a message for a specific threshold". Is this a new task? Because we use the name of the trade pair to create the message